bin/make-test

```diff
@@ -1,10 +1,12 @@
-#! /bin/sh
+#!/usr/bin/env bash
+
+set -eu -o pipefail
 
 go_test_output="/tmp/go-test.out"
 
 go test -v github.com/trussworks/truss-aws-tools/... | tee "${go_test_output}"
 
-if [ -n "$CIRCLECI" ]; then
+if [[ -n ${CIRCLECI-} ]]; then
```

Comment:
Question: What is this testing? I was thinking it is for detection if in a CircleCI environment, but I'm not following which envar is set to have a `-` vs an `_`.

Reply:
It is detecting a CircleCI environment, see the comment above for more details.